src/Testing/PHPUnit/AbstractGenericRectorTestCase.php

```diff
@@ -192,11 +197,6 @@ protected function getCurrentTestRectorClassesWithConfiguration(): array
         return [$rectorClass => null];
     }
 
-    /**
-     * Return interface type that extends @see \Rector\Core\Contract\Rector\RectorInterface;
-     */
-    abstract protected function getRectorInterface(): string;
-
     protected function yieldFilesFromDirectory(string $directory, string $suffix = '*.php.inc'): Iterator
```

Comment:
I like this. It gives the information, what needs to be implemented by children, much sooner.

:+1: